.github/workflows/test.yml

```diff
@@ -62,14 +62,12 @@ jobs:
     strategy:
       matrix:
         scenario:
-          - ember-lts-2.8
-          - ember-lts-2.12
-          - ember-lts-2.16
-          - ember-lts-3.4
-          - ember-lts-3.8
+          - ember-lts-3.16
+          - ember-lts-3.20
```

Comment:
@ivanvotti thoughts? I think this is okay personally given that many of these aren't even receiving support from Ember Core these days.

Reply:
That's fine. Though, it would be good to add a note about this to the Readme.

Reply:
https://github.com/ivanvotti/ember-svg-jar/blob/master/README.md#-compatibility << i.e. this should be updated.

I think it ought to say "Any active Ember v3 LTS, or release."